io/disk/ssd/nvmetest.py

```diff
@@ -126,6 +127,127 @@ def reset_controller_sysfs(self):
             % self.device.split("/")[-1]
         return process.system(cmd, shell=True, ignore_status=True)
 
+    def get_max_ns_count(self):
+        """
+        Returns the maximum number of namespaces supported
+        """
+        output = self.get_id_ctrl_prop('nn')
+        if output:
+            return int(output)
+        return 1
+
+    def get_total_capacity(self):
+        """
+        Returns the total capacity of the nvme controller. 0 if not found.
+        """
+        output = self.get_id_ctrl_prop('tnvmcap')
+        if output:
+            return int(output)
+        return 0
+
+    def ns_list(self):
+        """
+        Returns the list of namespaces in the nvme controller
+        """
+        cmd = "nvme list-ns %s" % self.device
+        namespaces = []
+        for line in process.system_output(cmd, shell=True,
+                                          ignore_status=True).splitlines():
+            namespaces.append(int(line.split()[1].split(']')[0]) + 1)
+        return namespaces
+
+    def list_ns(self):
+        """
+        Prints the namespaces list command, and does a rescan as part of it
+        """
+        cmd = "nvme ns-rescan %s" % self.device
+        process.system(cmd, shell=True, ignore_status=True)
+        cmd = "nvme list"
+        return process.system_output(cmd, shell=True, ignore_status=True)
+
+    def get_ns_controller(self):
+        """
+        Returns the nvme controller id
+        """
+        cmd = "nvme list-ctrl %s" % self.device
+        output = process.system_output(cmd, shell=True, ignore_status=True)
+        if output:
+            return output.split(':')[-1]
+        return ""
+
+    def get_lba(self):
+        """
+        Returns LBA of the namespace, defaults to 0 if not found
```

Comment:
Here as well 
`Returns LBA of the namespace otherwise  return  defaults to 0 (not found)`